main.c

```diff
@@ -1012,9 +1012,11 @@ main(int argc, char **argv)
 	    SearchHeader = search_header;
 	    DefaultType = default_type;
 	    char *url;
+	    int retry = 0;
 
 	    url = load_argv[i];
 	    if (getURLScheme(&url) == SCM_MISSING && !ArgvIsURL)
+	  retry_as_local_file:
 		url = file_to_url(load_argv[i]);
 	    else
```

Comment:
Style nit: I'd put braces around it. Or maybe something like the following:

```suggestion
	  retry_as_local_file:
	    if (retry || (getURLScheme(&url) == SCM_MISSING && !ArgvIsURL))
		url = file_to_url(load_argv[i]);
	    else
```